.github/ISSUE_TEMPLATE/000-report-crash.yml

```diff
@@ -72,7 +72,9 @@ body:
   id: report
   attributes:
     label: Crash Report
+    value: <details><summary>Log</summary>
     description: "If no crash report is created, attach `logs/fml-client-latest.log` or `logs/fml-server-latest.log`, depending on if it's a dedicated server crash or a client crash. You can also type in additional info that you consider important."
+    value: </details>
```

Comment:
You are not allowed to have duplicate keys in YAML, unfortunately.

Reply:
So this won’t work then…